### PR TITLE
add skipBlanks support for dz and zoomify layout

### PR DESCRIPTION
**Pull request by RaboliotTheGrey**
Hi!

Here it is ! 

* I didn't upgrade the libvips in the package.json version
* It seems that upgrade libvips to 8.8.0 broke 2 tests (`Image metadata ->  JPEG with IPTC/XMP` and `Attention strategy -> JPEG`) It also appears with master when libvips 8.8.0 is activated. I didn't fix them because i don't underestand their purposes ...
* I had to force the the skipBlanks in output.js because `AttrTo<int32_t>(options, "skipBlanks")` return 0 if it's NULL/undefined

Let me know if it needs some modifications.

### CODE REVIEW

**Comment by lovell** [comment]
Bonjour Jordan, merci beaucoup for this PR, which from a quick scan looks good - thank you for adding tests!

I notice there's some logic in libvips that defaults `skip_blanks` to a value of 5 when using the "google" layout, which this change should probably mirror.

https://github.com/libvips/libvips/blob/bcdaeca57877ba1d8c3cc7659708e4ef6f396a0b/libvips/foreign/dzsave.c#L1832-L1836

This PR will have to wait until after libvips v8.8.0 is released, which sharp will probably pick up in v0.23.0. I'll fix up the tests at that point to reflect the new/changed features that are making them fail.

**Comment by RaboliotTheGrey** [comment]
Hi, I've fixes this part and added a test to the google layout. 

let me know if you see any improvement :)

**Comment by lovell** [comment]
@RaboliotTheGrey I've changed the destination branch for this PR to the `vision` branch, which will become v0.23.0. Are you able to rebase against the `vision` branch and push to allow the CI tests to pass?

**Comment by RaboliotTheGrey** [comment]
Thanks!
I wont be able to rebase it until thuesday. :)
Have a nice week-end

-------- Original Message --------

**Comment by RaboliotTheGrey** [comment]
Hi,

I've rebase the branch on the lastest version of vision and test agains the version 8.8.1 of libvips.
All the tests are green on local.

File: lib/output.js [hunk 1]
```diff
@@ -539,6 +539,7 @@ function toFormat (format, options) {
  * @param {Number} [tile.overlap=0] tile overlap in pixels, a value between 0 and 8192.
  * @param {Number} [tile.angle=0] tile angle of rotation, must be a multiple of 90.
  * @param {String} [tile.depth] how deep to make the pyramid, possible values are `onepixel`, `onetile` or `one`, default based on layout.
+ * @param {Number} [tile.skipBlank=-1] threshold to skip tile generation, a value 0 - 255 for 8-bit images or 0 - 65535 for 16-bit images
```
Comment (lovell) [inline lib/output.js]: I think `tile.skipBlank` should be the plural form `tile.skipBlanks`.

Please can you run `npm run docs` after fixing this, which should automagically update `docs/api-output.md`.

Comment (RaboliotTheGrey) [reply lib/output.js]: Nice catch :)
Sorry I took a bit long to fix this.

**Comment by lovell** [comment]
Merveilleux, merci beaucoup.

**Pull request merged**